src/extends/index.d.ts

```diff
@@ -10,6 +10,7 @@ import type { VueExtensions } from './eslint-plugin-vue';
 import type { VuePugExtensions } from './eslint-plugin-vue-pug';
 import type { IntlifyVueI18nExtensions } from './intlify-vue-i18n';
 import type { TypescriptEslintExtensions } from './typescript-eslint';
+import type { SonarjsExtensions } from './eslint-plugin-sonarjs';
```

Comment:
I think you can fix things like these with VSCode Organize Imports

Reply:
Okay actually it's via a prettier plugin that you sort the imports. But in my case it doesn't seem to want to use it. I'll try to investigate